styles/main.css

```diff
@@ -1,283 +1,314 @@
   font-style: normal;
   font-weight: 400;
 }
-.main-nav-list{
+.main-nav-list {
   margin-left: 0;
   margin-right: 0;
   padding-left: 0;
   padding-right: 0;
   list-style-type: none;
 }
-.main-nav-list-item{
+.main-nav-list-item {
   list-style-type: none;
   display: inline-block;
-  margin-left: 15px;
+  padding-left: 15px;
 }
-.numbers{
+.numbers {
+  display: block;
   margin-top: 30px;
   margin-bottom: 10px;
   font-weight: bold;
   font-size: 100px;
   color: rgba(16, 16, 16, 0.08);
 }
-.features-link-text{
-  color: #ED4C5C;
+.features-link-text {
+  color: #ed4c5c;
 }
-.image-technology{
+.image-technology {
+  text-align: center;
+  min-height: 70px;
   margin-top: -80px;
 }
-.image-features{
+.image-features {
   width: 100%;
 }
-.text-features{
+
+.text-features {
+  color: #fff;
   padding-top: 16px;
   padding-bottom: 16px;
 }
-.main-map{
+.main-map {
   height: 310px;
   width: 100%;
   border-radius: 8px;
 }
-.main-nav a{
+.main-nav a {
   color: #fff;
   text-transform: uppercase;
 }
-.button-login:hover{
+.button-login:hover {
   background-color: #fff;
   color: #000;
 }
-.button-login:active {
+.button-login:active  {
   color: #7cc9f0;
-  background-color: #555555;
+  background-color: #555;
   }
-.button-login:focus{
+.button-login:focus {
   outline: solid #008000;
 }
-.video{
-  width: 400px;
+.logo-platform {
+  padding-right: 20px;
+}
+.video {
+  max-width: 100%;
   height: 311px;
 }
-.button-signup{
+.button-signup {
+  text-transform: uppercase;
+  margin-top: 40px;
   padding: 15px 45px;
   background-color: #fff;
-  color: #E63244;
+  color: #e63244;
   border-radius: 6px;
+  border: none;
 }
-.button-signup:active{
-  background-color: #E63244;
+.button-signup:active {
+  background-color: #e63244;
   color: #000;
 }
-.button-signup:hover{
-  background-color: #E63244;
+.button-signup:hover {
+  background-color: #e63244;
   color: #000;
 }
-.button-signup:focus{
+.button-signup:focus {
   outline: solid #008000;
 }
-  .bg-image{
+  .bg-image {
   background-color: #c7b39b;
   background-image: url("../images/firstimage.jpg"); 
   width: 1440px;
   height: 782px;
 } 
-.desccloud{
+.desccloud {
+  padding-left: 30px;
+  padding-right: 30px;
   text-align: center;
   font-size: 24px;
   font-style: normal;
-  font-weight: 700;
+  font-weight: 400;
   color:#fff;
 }
-.text-video{
+.text-video {
   text-align: justify;
   font-size: 16px;
   font-style: normal;
   font-weight: 400;
 }
-.text-center{
+.text-center {
   text-align: center;
 }
-.first-title{
-  margin-top: 170px;
+.first-title {
+  margin-top: 250px;
   font-weight: bold;
   font-size: 72px;
   text-transform: uppercase;
   color: #fff;
 }
-.title-medium{
+.title-medium {
   font-style: normal;
   font-weight: 700;
   font-size: 36px;
   line-height: 43px;
   color: #000;
 }
-.button-play{
+.button-invideo {
+  position: relative; 
+}
+.button-play {
+  position: absolute;
+  left: calc(50% - 10px);
+  top: calc(50% - 10px);
   border: none;
```

Comment:
I guess we need add width and height for the button. Check styles according to design.